README.md

```diff
@@ -1,11 +1,11 @@
 # 📦 krates
 
-[![Build Status](https://github.com/EmbarkStudios/krates/workflows/CI/badge.svg)](https://github.com/EmbarkStudios/krates/actions?workflow=CI)
+[![Embark](https://img.shields.io/badge/embark-open%20source-blueviolet.svg)](https://embark.dev)
+[![Embark](https://img.shields.io/badge/discord-ark-%237289da.svg?logo=discord)](https://discord.gg/dAuKfZS)
 [![Crates.io](https://img.shields.io/crates/v/krates.svg)](https://crates.io/crates/krates)
 [![Docs](https://docs.rs/krates/badge.svg)](https://docs.rs/krates)
-[![Rust Version](https://img.shields.io/badge/Rust%20Version-1.47.0-blue.svg)](https://forge.rust-lang.org/release/platform-support.html)
```

Comment:
This badge should stay, it's what tells you what rust version this crate has been updated to handle the builtin target triples in rustc.

Reply:
Can do. Is this another one we want to discuss in https://github.com/EmbarkStudios/opensource-template/issues/20 ? Seems like it would be useful everywhere if it is useful here.

Reply:
No, this isn't the MSRV like with cargo-deny, just what up to what version rustc this crate "understands".

Reply:
OK done! ty